src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -444,4 +445,66 @@
         </xsd:sequence>
         <xsd:attribute name="id" type="xsd:string" use="required"/>
     </xsd:complexType>
+
+    <xsd:complexType name="http_client">
+        <xsd:sequence>
+            <xsd:element name="max_host_connections" type="xsd:integer" minOccurs="0" />
+            <xsd:element name="no_proxy" type="xsd:string" minOccurs="0" />
+            <xsd:element name="default_options" type="http_client_options" minOccurs="0" />
+            <xsd:element name="http_client" type="http_client_client" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="api_client" type="http_client_client" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:attribute name="enabled" type="xsd:boolean" />
+    </xsd:complexType>
+
+    <xsd:complexType name="http_client_options">
+        <xsd:sequence>
+            <xsd:element name="auth" type="xsd:string" minOccurs="0" />
```

Comment:
should we sort these options alphabetically? I don't know what's the Symfony policy about this in XSD files.

Reply:
Keeping the order from HttpClientInterface looks consistent to me